metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bigcommerce-monit",
+  "version": "1.0.0",
+  "author": "Bigcommerce - Technical Operations <techops@bigcommerce.com>",
+  "license": "Proprietary",
+  "summary": "BigCommerce Monit Module - Manages monit on the BigCommerce platform",
+  "source": "https://github.com/bigcommerce/puppet-module-monit",
+  "project_page": "https://github.com/bigcommerce/puppet-module-monit",
+  "tags": [
+    "bigcommerce",
```

Comment:
Minor nitpick, I think you should remove `bigcommerce` from the tags since this module isn't specific to bigcommerce besides it being hosted in our github org.